Tests.Integration.Presentation.Web/Interfaces/V2/ItInterfaceApiV2Test.cs

```diff
@@ -430,6 +430,26 @@ public async Task GET_Many_As_Stakeholder_With_ChangesSince_Filter()
             Assert.Equal(2, dtos.Count);
             Assert.Equal(new[] { itInterface3.Uuid, itInterface1.Uuid }, dtos.Select(x => x.Uuid).ToArray());
         }
+        
+        [Fact]
+        public async Task GET_Many_As_Stakeholder_With_Name_Filter()
+        {
+            //Arrange
+            var (token, organization) = await CreateStakeHolderUserInNewOrg();
+            var system = await ItSystemHelper.CreateItSystemInOrganizationAsync(A<string>(), TestEnvironment.DefaultOrganizationId, AccessModifier.Public);
+            var searchName = A<string>();
+            var itInterface1 = await InterfaceHelper.CreateInterface(InterfaceHelper.CreateInterfaceDto(searchName, A<string>(), TestEnvironment.DefaultOrganizationId, AccessModifier.Public));
+            await InterfaceHelper.CreateInterface(InterfaceHelper.CreateInterfaceDto(A<string>(), A<string>(), TestEnvironment.DefaultOrganizationId, AccessModifier.Public));
```

Comment:
if the second created interface does not have searchName as part of the name then we don't prove that it is by excact name and not content. Set the searchName as the prefix by simply `${searchName}1"`